components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt/src/main/java/org/wso2/carbon/identity/claim/metadata/mgt/dao/ClaimConfigInitDAO.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Use new license header
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org).
```